gcc-config

```diff
@@ -813,11 +813,12 @@ switch_profile() {
 	# Update LTO plugin for GCC.  Supported as of binutils 2.25.
 	local BFD_PLUGINS_DIR
 	local LIBLTO_PLUGIN
-	LIBLTO_PLUGIN="${EROOT}usr/libexec/gcc/${CTARGET}/${CC_COMP_VERSION}/liblto_plugin.so"
 	if is_cross_compiler; then
 		BFD_PLUGINS_DIR="${EROOT}usr/${CHOST}/${CTARGET}/binutils-bin/lib/bfd-plugins"
+		LIBLTO_PLUGIN="../../../../../libexec/gcc/${CTARGET}/${CC_COMP_VERSION}/liblto_plugin.so"
```

Comment:
I suppose we could/should drop in the same thing we use in [PMS](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-129018r2):
```
dosym_relative_path() {
    local link=$(realpath -m -s "/${2#/}")
    local linkdir=$(dirname "${link}")
    realpath -m -s --relative-to="${linkdir}" "$1"
}
```
then call that.